src/chapters/exceptions/studio.rst

```diff
@@ -0,0 +1,23 @@
+Strategic Debugging
+===================
+
+Summary
+-------
+
+At this point, we have seen a lot of different types of errors.
+We have possibly created logic errors or syntax errors and now, we have just learned about the ``Error`` object in JavaScript.
+The goal of this studio is for us to develop strategies for debugging so that we can get rid of the bugs and get back to coding!
+
+Activity
+--------
+
+Think of a bug you have seen in your code.
+This could be the time you dropped a keyword when initializing a variable or misused a method.
+
+1. Take some time to discuss with the group what your error was and how you solved it.
+   Did you talk to a TA to get it?
+   Did you find a great resource online that was helpful?
+
+Your TA will go over the pros and cons of different resources that can help you resolve the error.
```

Comment:
I think we should provide them with a basic list of places to go when they encounter an error.

1. Google the error
2. Look for examples on Stackoverflow
3. Look over the documentation
4. Talk to a coworker, mentor, or TA
5. Describe the problem in detail. ie what information is coming in, and what is the desired behavior or output.